index.html

```diff
@@ -327,8 +327,9 @@
       </section>
       <section id="subscription-response-details">
         <h4>Subscription Response Details</h4>
-        <p>The hub MUST respond to a subscription request with an HTTP [[!RFC7231]] 202 "Accepted" response to indicate that the request was received and will now be verified (<a href="#hub-verifies-intent">Section 4.3 </a>) and validated (<a href="#subscription-validation">Section 4.2 </a>) by the hub. The hub SHOULD perform the verification and validation of intent as soon as possible.</p>
+        <p>If the hub support WebSub and is able to handle the (un)subscription request, it MUST respond to a subscription request with an HTTP [[!RFC7231]] 202 "Accepted" response to indicate that the request was received and will now be verified (<a href="#hub-verifies-intent">Section 4.3 </a>) and validated (<a href="#subscription-validation">Section 4.2 </a>) by the hub. The hub SHOULD perform the verification and validation of intent as soon as possible.</p>
         <p>If a hub finds any errors in the subscription request, an appropriate HTTP [[!RFC7231]] error response code (4xx or 5xx) MUST be returned. In the event of an error, hubs SHOULD return a description of the error in the response body as plain text, used to assist the client developer in understanding the error. This is not meant to be shown to the end user. Hubs MAY decide to reject some callback URLs or topic URLs based on their own policies (e.g., domain authorization, topic URL port numbers). However, since verification and validation of intent are asynchronous steps that logically begin after the HTTP response has been returned, the HTTP response MUST NOT depend on the process or outcome of verification or validation.</p>
+        <p>If the hub is not able to handle (un)subscription requests, it MAY redirect to another hub which supports WebSub. It does so by yielding an HTTP [[!RFC7231]] 307 (temporary redirect) or 308 (permanent redirect) response. It MUST also include at least a HTTP [[!RFC7230]] Location Header containing a preferred URL reference for the hub to use by the subscriber.</p>
```

Comment:
"If the hub URL is not able to handle subscription or unsubscription requests, it MAY..."

then add:

"The subscriber is expected to retry the subscription or unsubscription at the new hub URL"